java/scripts/sign_and_bundle_release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ];
```

Comment:
Can you stick all this stuff in functions? I think just a `usage` and a `main` oughta do it but I don't love top-level commands in scripts (it leads to issues if you edit the script while it's running, for instance)